### ISSUE TITLE
Selenium do not work in docker 

### ISSUE THREAD

**Issue by NeoUKR** [Pullger/pullgerSquirrel#1]
Critical error on initialization driver.

**Comment by NeoUKR** [comment]
Need to add the following initial options for chrome:

```
        chrome_options.add_argument('--no-sandbox')
        chrome_options.add_argument('--window-size=1920,1080')
        chrome_options.add_argument('--disable-gpu')
```

**Comment by NeoUKR** [comment]
Fixed on release 0.2.2.0
